### ISSUE TITLE
deno compile execute error ReferenceError: normalize is not defined 

### ISSUE THREAD

**Issue by PaulThompson** [denoland/deno#9591]
Summary:
Some code which installs and runs successfully actually fails with an error when used with `deno compile`.

Details:
Trying out `deno compile` for task running tool project: https://deno.land/x/dnit
Using current deno version 1.7.5

```
$ deno compile --allow-read --allow-write --allow-run --unstable https://deno.land/x/dnit@dnit-v1.11.0/main.ts
Check https://deno.land/x/dnit@dnit-v1.11.0/main.ts
Bundle https://deno.land/x/dnit@dnit-v1.11.0/main.ts
Compile https://deno.land/x/dnit@dnit-v1.11.0/main.ts
Emit dnit

$ ./dnit
error: ReferenceError: normalize is not defined
    at file://$deno$/bundle.js:3749:20
```

The same code runs ok when used with `deno install`:
```
$ deno install --allow-read --allow-write --allow-run --unstable -f --name dnit https://deno.land/x/dnit@dnit-v1.11.0/main.ts
✅ Successfully installed dnit
/home/pault/.deno/bin/dnit
$ dnit --version
dnit 1.11.0
```

```$ deno --version
deno 1.7.5 (release, x86_64-unknown-linux-gnu)
v8 9.0.123
typescript 4.1.4
```

By the looks of it it's a duplicate of keywords/issues:
`String.prototype.normalize()`
https://github.com/denoland/deno/issues/1968

**Comment by kitsonk** [comment]
cc/ @kdy1

**Comment by IllusionPerdu** [comment]
Recently, they're many bug fix for bundler in swc. Probably deno should update the depency of swc

**Comment by kitsonk** [comment]
@IllusionPerdu we will and we do.

**Comment by kdy1** [comment]
Fixed on upstream.

**Comment by PaulThompson** [comment]
Fixed in deno 1.8.0, thanks.
